Functions.Templates/Templates/EventGridBlobTrigger-JavaScript/function.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
add the template to `/Build/PackageFiles/ExtensionBundle/ExtensionBundleTemplates-4.x.nuspec`